origin-growth/src/enums.js

```diff
@@ -20,6 +20,32 @@ const GrowthCampaignRewardStatuses = new Enum(
   'Distributed'
 )
 
+const GrowthCampaignStatuses = new Enum(
```

Comment:
nit: how should we capitalize those enum values ? I used upper case. But maybe I should have used lower case ? I don't really care one way or the other but maybe we should try to be consistent.

Reply:
Yeah I noticed that. I used lower case because these are directly the values used in `Growth.js` GraphQL typedef. I guess we could: 
1. make these upper case and call `toLowerCase()` before returning values in the resolvers. 
2. make these lowercase across the system. 

Do you have any preference? Perhaps 1. would be easier to implement

Reply:
I don't have any strong preference... We haven't run the DB schema in prod for growth so there is no operational overhead if we decide to go for option 2.
Do what you think is best :)